### PR TITLE
Sensorpush api refactor

### PR DESCRIPTION
**Pull request by alneberg**
Turned out I needed to change the script quite a lot. It is now designed to run hourly and to fetch each sensor individually, this generates more requests, but minimises the payload size from the API which is apparently the cause of the timeout issues I had.

The database document is still per 24 hours so the script can now handle a merge of the newly fetched data and the existing data in the database.

### CODE REVIEW

File: sensorpush_to_statusdb.py [hunk 1]
```diff
@@ -354,8 +455,18 @@ def main(
...
+        view_call = sensorpush_db.view("entire_document/by_name_and_date")[
+            sd.sensor_name, sd.start_date_midnight
+        ]
+
         sd_dict = sd.format_for_statusdb()
 
+        if view_call.rows:
+            sd_dict = SensorDocument.merge_with(sd_dict, view_call.rows[0].value)
+            sd_dict["_id"] = view_call.rows[0].value["_id"]
+            sd_dict["_rev"] = view_call.rows[0].value["_rev"]
```
Comment (aanil) [inline sensorpush_to_statusdb.py:468]: Why not put these in the `merge_with` method? It gets all of `view_call.rows[0].value` anyway.

**Review by aanil — APPROVED** [review]
Code looks ok, eventhough I'm not too sure of the functionality!

File: sensorpush_to_statusdb.py (hunk 1 above)
Comment (alneberg) [reply sensorpush_to_statusdb.py:468]: Yes! I will update that.

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**